devops/linux/docker/helper/scripts/d-stack-up-app-containers.sh

```diff
@@ -26,12 +26,6 @@ pushd .
 
   echo
 
-  pushd helper/keycloak
```

Comment:
Hi @sushantpatil1214 . I am just going to revert this for now, as all the old scripts should work as before, until we have fully tested the new scripts. Currently this would break the `devops/linux/docker/d-stack-3-reboot.sh`